sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateOctKeyOptions.java

```diff
@@ -5,21 +5,51 @@
 import java.time.OffsetDateTime;
 import java.util.Map;
 
+/**
+ * Represents the configurable options to create an AES key.
+ */
 public class CreateOctKeyOptions extends CreateKeyOptions {
+    /**
+     * The AES key size.
+     */
+    private Integer keySize;
+
     /**
      * The hardware protected indicator for the key.
      */
     private boolean hardwareProtected;
 
     /**
-     * Creates a CreateOctKeyOptions with {@code name} as name of the Rsa key.
+     * Creates a {@link CreateOctKeyOptions} with {@code name} as name of the AES key.
      *
      * @param name The name of the key.
      */
     public CreateOctKeyOptions(String name) {
         super(name, KeyType.OCT);
     }
 
+    /**
+     * Get the key size in bits.
+     *
+     * @return The key size in bits.
+     */
+    public Integer getKeySize() {
+        return this.keySize;
```

Comment:
Whether this should be calling into the parent or not, I don't see getters for other properties. Shouldn't there be? In .NET, we almost never have write-only properties: just read-only or read-write.